CMakeLists.txt

```diff
@@ -126,24 +150,24 @@ add_library(rdf4cpp
         src/rdf4cpp/rdf/writer/NNodeWriter.cpp
         src/rdf4cpp/rdf/writer/NQuadsWriter.cpp
         src/rdf4cpp/rdf/writer/NTriplesWriter.cpp
-        ${serd_source_files}
-        )
+        ${serd_source_files})
+
```

Comment:
more a style comment: I prefer here trailing `)` in a new line for nicer git diffs.